src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/induction/InductionEntityListManager.kt

```diff
@@ -0,0 +1,67 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.mapper.induction
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.induction.EntityKeyAware
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.DomainKeyAware
+
+/**
+ * Provides a centralised place to update, add or remove existing JPA entities (and thereby apply changes to the
+ * database). Manages changes between lists of domain model objects (provided via the REST API) and the equivalent
+ * lists of existing JPA entities.
+ */
+@Component
+class InductionEntityListManager<ENTITY : EntityKeyAware, DOMAIN : DomainKeyAware> {
```

Comment:
I haven't provided a test for the class as it's tested indirectly as part of the many mapper tests.